v2/bench_test.go

```diff
@@ -2,12 +2,20 @@ package car_test
 
 import (
 	"io"
+	"math/rand"
 	"os"
+	"path/filepath"
 	"testing"
 
+	"github.com/ipfs/go-cid"
+	"github.com/ipfs/go-merkledag"
+	"github.com/ipld/go-car/v2/blockstore"
+
 	carv2 "github.com/ipld/go-car/v2"
 )
 
+var rng = rand.New(rand.NewSource(1413))
```

Comment:
nit: move this inside generateRandomCarV2File, because then each call to that function will be deterministic and produce the same file. otherwise running the benchmarks in a different order (or only running one of them) can cause different behavior and resulting numbers.

plus, fewer globals.